Source/ConfigCat/Private/ConfigCatSubsystem.cpp

```diff
@@ -279,7 +284,12 @@ void UConfigCatSubsystem::Initialize(FSubsystemCollectionBase& Collection)
 	const UConfigCatSettings* ConfigCatSettings = GetDefault<UConfigCatSettings>();
 	if (!ConfigCatSettings || ConfigCatSettings->SdkKey.IsEmpty())
 	{
-		UE_LOG(LogConfigCat, Warning, TEXT("Empty SdkKey detected. Please set your SdkKey in the Project Settings."));
+		if(GEngine)
+		{
+			GEngine->AddOnScreenDebugMessage(-1, 5.0f, FColor::Red, TEXT("ConfigCat SdkKey missing. Please set your SdkKey in the Project Settings."));
```

Comment:
Could you please move the error message into a variable to avoid string duplication?

Reply:
It's refactored now into a `PrintError` function for both cases.